Makefile

```
@@ -125,9 +125,11 @@ generate-keyboards-file:
 	exit 0
 
 clean:
```

```suggestion
.PHONY: clean
clean:
```
Do this for `.PHONY: list-keyboards` (line 115), `.PHONY: generate-keyboards-file` (line 123) and `.PHONY: git-submodule` (line 582) as well.